README.md

```diff
@@ -1,12 +1,8 @@
-# Exploring_Hacker_News_Posts
+# Analyzing Hacker News Posts
```

Comment:
I just made this consistent with the title used in your notebook and I removed the underscores since a README is more of a human-friendly document and this should be a readable title.

You might also consider renaming the ipynb file to use the same name.